tracpro/groups/forms.py

```diff
@@ -18,6 +22,8 @@ def __init__(self, model, org, *args, **kwargs):
         # Retrieve Contact Group choices from RapidPro.
         choices = [(group.uuid, "%s (%d)" % (group.name, group.size))
                    for group in self.org.get_temba_client().get_groups()]
+        # Sort choices by the labels
+        choices.sort(key=itemgetter(1))
```

Comment:
I'm not exactly sure why but this is only sorting some of the items correctly. A few extra items get appended onto the end of the list:
![image](https://cloud.githubusercontent.com/assets/11393311/23380271/1a33b58c-fd08-11e6-99c6-6e104dd29fc3.png)